Documentation/op-guide/etcd2.rules

```diff
@@ -0,0 +1,121 @@
+### General cluster availability ###
+
+# alert if another failed peer will result in an unavailable cluster
+ALERT InsufficientPeers
+  IF count(up{job="etcd-k8s"} == 0) > (count(up{job="etcd-k8s"}) / 2 - 1)
+  FOR 3m
+  LABELS {
+    severity = "critical"
+  }
+  ANNOTATIONS {
+    summary = "Etcd cluster small",
+    description = "If one more etcd peer goes down the cluster will be unavailable",
+  }
+
+### HTTP requests alerts ###
```

Comment:
etcd3's main request metrics should be gRPC. let's replace this with gRPC alters